desc/objectives/_stability.py

```diff
@@ -367,3 +369,278 @@ def compute(self, params, constants=None):
             profiles=constants["profiles"],
         )
         return constants["transforms"]["grid"].compress(data["magnetic well"])
+
+
+class BallooningStability(_Objective):
+    """A type of ideal MHD instability.
+
+    Infinite-n ideal MHD ballooning modes are of significant interest.
+    These instabilities are also related to smaller-scale kinetic instabilities.
+    With this class, we optimize MHD equilibria against the ideal ballooning mode.
+
+    Targets the following metric:
+
+    f = w₀ sum(ReLU(γ-γ₀)) + w₁ max(ReLU(γ-γ₀))
+
+    where γ is the negative squared growth rate for each field line (such that γ>0 is
```

Comment:
Another general comment - For the other stability objectives (`MercierStability`, `MagneticWell`) we've chosen them so that `metric > 0` is stable, (this is also the case for the newcomb metric) while this is the opposite. Do we want to throw on a minus sign to be consistent? or is that more confusing since gamma is sort of a growth rate, which suggests positive values are unstable?

Reply:
The current convention is chosen to be consistent with COBRAVMEC. I don't think we should change that.